.ci/build.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  python_versions : []
+
+steps:
+  - ${{ each py_version in parameters.py_versions }}:
+    - task: usePythonVersion@0
+      inputs:
+        versionSpec: '3.6'
+
+    - checkout: self
+      clean: true
+
+    - script: python3 -m pip install --user pyyaml
+      displayName: 'Install PyYaml'
+
+    - script: python3 jellyfin-kodi/.config/generate_xml.py ${{ py_version }}
+      displayName: 'Create ${{ py_version }} addon.xml'
+
+    - script: zip -r plugin.video.jellyfin-${{ py_version }}.zip jellyfin-kodi
```

Comment:
Use this: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/archive-files?view=azure-devops